docs/config.md

```diff
@@ -0,0 +1,124 @@
+                                                         # The following env vars needs to be set: SLACK_API_TOKEN ( Bot User OAuth Access Token ) and SLACK_CHANNEL ( channel to send notifications in case of failures )
+                                                         # When slack_integration is enabled, a watcher can be assigned for each day. The watcher of the day is tagged while reporting failures in the slack channel. Values are slack member ID's.
+    watcher_slack_ID:                                        # (NOTE: Defining the watcher id's is optional and when the watcher slack id's are not defined, the slack_team_alias tag is used if it is set else no tag is used while reporting failures in the slack channel.)
+        Monday:
+        Tuesday:
+        Wednesday:
+        Thursday:
+        Friday:
+        Saturday:
+        Sunday:
+    slack_team_alias:                                    # The slack team alias to be tagged while reporting failures in the slack channel when no watcher is assigned
+
+    custom_checks:
+        -   custom_checks/custom_check_sample.py       # Relative paths of files conataining additional user defined checks
+
+tunings:
+    timeout: 20                                          # Number of seconds before requests fail
+    iterations: 1                                        # Iterations to loop before stopping the watch, it will be replaced with infinity when the daemon mode is enabled
+    sleep_time: 3                                       # Sleep duration between each iteration
+    kube_api_request_chunk_size: 250                     # Large requests will be broken into the specified chunk size to reduce the load on API server and improve responsiveness.
+    daemon_mode: True                                    # Iterations are set to infinity which means that the cerberus will monitor the resources forever
+    cores_usage_percentage: 0.5                          # Set the fraction of cores to be used for multiprocessing
+
+database:
+    database_path: /tmp/cerberus.db                      # Path where cerberus database needs to be stored
+    reuse_database: False                                # When enabled, the database is reused to store the failures
+```
+
+#### Watch Nodes
+This flag returns any nodes where the KernelDeadlock is not set to False and does not have a `Ready` status
+
+#### Watch Cluster Operators
+When `watch_cluster_operators` is set to True, this will monitor the degraded status of all the cluster operators and report a failure if any are degraded
+False will not query or report the status of the cluster operators
+
+
+#### Watch Routes
+This parameter expects a double array with each item having the url and an optional bearer token or authorization for each of the url's to properly connect
+
+For example:
+```
+watch_url_routes:
+- - <url>
+  - <authorization> (optional)
+- - https://prometheus-k8s-openshift-monitoring.apps.****.qe.devcluster.openshift.com
```

Comment:
Just a small nit, I'd suggest changing the domains to be as generic as possible:
`https://prometheus-k8s-openshift-monitoring.apps.****.cluster.openshift.com`